src/ripple/app/misc/HashRouter.h

```diff
@@ -68,50 +68,42 @@ class HashRouter
         {
         }
 
-        std::set <PeerShortID> const& peekPeers () const
-        {
-            return peers_;
-        }
-
         void addPeer (PeerShortID peer)
         {
             if (peer != 0)
                 peers_.insert (peer);
         }
 
-        bool hasPeer (PeerShortID peer) const
-        {
-            return peers_.count (peer) > 0;
-        }
-
         int getFlags (void) const
         {
             return flags_;
         }
 
-        bool hasFlag (int mask) const
-        {
-            return (flags_ & mask) != 0;
-        }
-
         void setFlags (int flagsToSet)
         {
             flags_ |= flagsToSet;
         }
 
-        void clearFlag (int flagsToClear)
+        void swap (std::set <PeerShortID>& other)
         {
-            flags_ &= ~flagsToClear;
+            peers_.swap (other);
         }
 
-        void swapSet (std::set <PeerShortID>& other)
+        bool prepRelay (Stopwatch::time_point const& now,
```

Comment:
`shouldRelay` sounds good, and I'll add some doc particularly to make clear that there are side effects. `touch` is too abstract, since there's other data in the object that can be updated independent of the relayability.